applications/app/views/fragments/gallerySlot.scala.html

```diff
@@ -12,7 +20,7 @@
         @fragments.commercial.adSlot(
             slotName,
             Seq("gallery-inline", "dark") ++ (if(isMobile) Some("mobile") else None),
-            Map("mobile" -> sizes),
+            Map("mobile" -> (sizes ++ additionalSizes)),
```

Comment:
Good point - I think `desktop` should suffice as that breakpoint min width is 980...